Update docker-basics-dotnet-core.md

jahbenjah opened the pull request:
## Summary
Update docker installation links
Updtate from dotnet 2.0 to 2.1
Fixes #Issue_Number (if available)

jahbenjah committed:
Update docker-basics-dotnet-core.md

Update docker installation links
Updtate from dotnet 2.0 to 2.1

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=8039) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=8039) before we can accept your contribution.<br/><br/>:x: jahbenjah [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=8039)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=8039) it.</sub>

kendrahavens reviewed (approved):
LGTM! Thank you!

 A small nit, referring to Docker "CE" might confuse users who aren't familiar with the Docker client offerings, but it's not a big issue.

jahbenjah committed:
Use Docker client instead of Docker CE

Change "Docker CE" by "Docker client"

kendrahavens reviewed (approved):
Thanks!

mairaw commented on docs/core/docker/docker-basics-dotnet-core.md:
```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
+#### .NET Core 2.1 SDK
 
-* Install [.NET Core SDK 2.0](https://www.microsoft.com/net/core).
+* Install [.NET Core SDK 2.1](https://www.microsoft.com/net/core).
```
wondering if sending them straight to downloads wouldn't be better:
https://www.microsoft.com/net/download

mairaw commented on docs/core/docker/docker-basics-dotnet-core.md:
```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
 
```
Can you also remove lines 4 and 5 (author and ms.author metadata) and update ms.date to 10/11/2018?

jahbenjah replied:
Done

mairaw reviewed (commented):
This looks great @jahbenjah. I left a couple of comments for you to consider.

jahbenjah committed:
Update docker-basics-dotnet-core.md

Update link to .NET  downloads
Remove author and ms.author
Update link to Visual Studio Code downloads

Thraka commented:
@mairaw I think this is good to go. I don't want to just approve it since you did have feedback. I do think he has addressed your feedback though.

mairaw reviewed (approved):
Yes @Thraka! Thanks for bringing this my attention. Thank you @jahbenjah for your contribution.
